vllm/config.py

```diff
@@ -107,6 +108,8 @@ class ModelConfig:
             can not be gathered from the vllm arguments.
         config_format: The config format which shall be loaded.
             Defaults to 'auto' which defaults to 'hf'.
+        bert_config: tokenizationconfiguration dictionary for a given 
```

Comment:
`bert_config` is not actually a parameter of the `__init__` method. The documentation can be moved close to the initialization of `self.bert_config` or removed. Otherwise the docstring will be wrong.